src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$hand-transform-origin: bottom center;
+$hand-bottom: 50%;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: dotted #000 1vmin;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the border property should be written in shorthand form if possible. In this case, 'dotted' and '#000' are the border-style and border-color values, but '1vmin' is the border-width for all four sides, so it should be written as 'border: 1vmin dotted #000;'.